### DISCUSSION TITLE
Question about the `asset tag`

### DISCUSSION THREAD

**Discussion by cymqqqq** [RGB-WG/rgb#225] [Q&A] [answered]
Why do we need an `asset tag` when we add a fungible asset? What about its design principle?

**Comment by dr-orlovsky** [comment, marked as answer]
This comes from the foundational [work on confidential assets](https://blockstream.com/bitcoin17-final41.pdf?ref=blog.blockstream.com) by Blockstream and is required for Liquid compatibility.
